### PR TITLE
Pneumatics Fix

### PR DESCRIPTION
**Pull request by ManiacMoustache**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Steps to Test
[ Describe how to run your AUTOMATED TESTS and any manual tests that may be need to prove your fix/feature REMOVE THIS WHEN DONE]
1. Get bot with compressor and solenoid
2. If compressor and solenoid works, you good

## Changelog
Added code to enable compressor

## What?
Updated initialization methods

## Why?
We can use pistons

#77

### CODE REVIEW

**Comment by ChrisHirsch** [comment]
Is there an issue associated with this PR?

File: utils/acturatorFactories.py [hunk 1]
```diff
@@ -23,19 +23,19 @@ def solenoidFactory(descp):
...
-    except Exception as e:
-        logging.error("Failed to create solenoid %s. Err %s", descp, e)
+    # try:
+    pcm = 0
+    if pcm in descp:
+        pcm = descp["pcm"]
+    if descp["type"] == "solenoid":
+        return wpilib.Solenoid(pcm, wpilib.PneumaticsModuleType.CTREPCM, descp["channel"])
+    if descp["type"] == "doubleSolenoid":
+        solenoid = wpilib.DoubleSolenoid(pcm, wpilib.PneumaticsModuleType.CTREPCM, descp["channel"]["forward"], descp["channel"]["reverse"])
```
Comment (ChrisHirsch) [inline utils/acturatorFactories.py:33]: Are sure this should be CTREPCM and not REVPH?

**Review by hell0w0rld2 — APPROVED** [review]
👍

**Review by hell0w0rld2 — APPROVED** [review]
good

**Review by Leaf-life — APPROVED** [review]
good

**Pull request merged**